File: 백준/Bronze/25304. 영수증/영수증.go
package main

import (
    "fmt"
    "bufio"
    "os"
)

func main() {
    reader := bufio.NewReader(os.Stdin)

    var x, n, a, b int
    
    fmt.Fscanf(reader, "%d\n", &x)
    fmt.Fscanf(reader, "%d\n", &n)

    for i := 0; i < n; i++ {
        fmt.Fscanf(reader, "%d %d\n", &a, &b)

        x -= a * b
    }

    answer := "No"
    if x == 0 {
        answer = "Yes"
    }

    fmt.Println(answer)
}

File: 백준/Bronze/25304. 영수증/영수증.c
#include <stdio.h>

int main() {
    int x = 0, n = 0, a = 0, b = 0;
    char *answer;

    scanf("%d\n", &x);
    scanf("%d", &n);

    for (int i = 0; i < n; i++) {
        scanf("%d %d", &a, &b);

        x -= a * b;
    }

    answer = x == 0 ? "Yes" : "No";

    printf("%s", answer);

    return 0;
}